src/matrixvariate.jl

```diff
@@ -175,10 +179,18 @@ getmatrix(p::PDMat) = p.mat
 getmatrix(p::PDiagMat) = Diagonal(p.diag)
 getmatrix(p::ScalMat) = Diagonal(fill(p.value, p.dim))
 
-function TuringInverseWishart(df::T, Ψ::AbstractMatrix) where T<:Real
+function TuringInverseWishart(df::Real, Ψ::AbstractMatrix)
+    C = cholesky(Ψ)
+    return TuringInverseWishart(df, Ψ, C)
+end
+function TuringInverseWishart(df::Real, D::AbstractPDMat)
+    Ψ = getmatrix(D)
+    C = getchol(D)
+    return TuringInverseWishart(df, Ψ, C)
+end
+function TuringInverseWishart(df::T, Ψ::AbstractMatrix, C::Cholesky) where T<:Real
```

Comment:
```suggestion
getmatrix(x::AbstractMatrix) = x

function TuringInverseWishart(df::T, x::AbstractMatrix) where T<:Real
    Ψ = getmatrix(x)
    C = getchol(x)
```
BTW it seems one could drop the `where` clause and just use `typeof(df)` in the function body?

Reply:
Alternatively (see above), we could maybe use
```suggestion
getmatrix(x::AbstractMatrix) = x

function TuringInverseWishart(df::T, x::AbstractMatrix) where T<:Real
    Ψ = getmatrix(x)
    C = cholesky(x)
```